CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.x]
+### Added
+- Added static `setFormats` and `addFormat` methods to `BaseSerializerHandler` class
```

Comment:
Yup. The goal is to avoid everyone editing the same line, creating conflicts again and again.